executable/swf.ksy

```diff
@@ -4,7 +4,10 @@ meta:
   file-extension: swf
   xref:
     justsolve: SWF
-    loc: fmt/507
+    pronom: fmt/507
```

Comment:
Actually, this error gets highlighted when validating the KSY against the [`ksy_schema.json`](https://github.com/kaitai-io/ksy_schema/blob/master/ksy_schema.json), but of course, someone would need to do that for all formats 😃:

![image](https://user-images.githubusercontent.com/47499687/100477752-99284b00-30e9-11eb-95c7-e3b473a51d94.png)

Perhaps we could enforce all specs in KSF to pass through the https://github.com/kaitai-io/ksylint tool, but we need to update the built-in KSY schema in it and possibly adding additional checks to the tool (for instance, if the resource at URL _`https://www.nationalarchives.gov.uk/pronom/` + value of the `meta/xref/pronom` key_ exists, i.e. checking if ttps://www.nationalarchives.gov.uk/pronom/fmt/507 returns proper HTTP 200 code) etc.

Reply:
Yeah, I found it that way.